mne/utils/docs.py

```diff
@@ -33,6 +33,64 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 
 docdict = dict()
 
+
+# %%%%
```

Comment:
In the tutorials and examples, we just do two of these `%%` so it would be nice to stick with that convention